dev/tools/merge-pr.sh

```diff
@@ -43,7 +43,7 @@ fi;
 git merge -S --no-ff FETCH_HEAD -m "Merge PR #$PR: $(curl -s "$API/pulls/$PR" | jq -r '.title')" -e
 
 # TODO: improve this check
-if [[ $(git diff "$REMOTE/$CURRENT_LOCAL_BRANCH" dev/ci) ]]; then
+if git diff --quiet "$REMOTE/$CURRENT_LOCAL_BRANCH" -- dev/ci; then
```

Comment:
I think you need to invert the logic by adding a `!` in front of `git diff`, like I did in #6565.

Reply:
Indeed.